### ISSUE TITLE
Getting Accessibility issues on Cookies page

### ISSUE THREAD

**Issue by Shalini-git25** [DFE-Digital/find-information-products-services-frontend#173]
**Describe the bug**
Getting below accessibility issues on Cookies page -
1. When running ARC Toolkit on page - showing error as 'Multiple main landmarks' for the page.
2. When using WAVE evaluation tool - showing Contrast errors as '2 X Very low contrast' for On and Off radio buttons under Essential cookies.

**To Reproduce**
1. Click on Cookies link present in Footer on any page.
2. Cookies page opens.
3. Run tests using ARC Toolkit on the page - error message is displayed as 'Multiple main landmarks'.
4. Now enable WAVE evaluation tool on the page.
5. Error is displayed under Details as '2 Contrast errors' for radio buttons.

**Expected behavior**
No accessibility error should be present on the Cookies page.

**Screenshots**
Accessibility Issue 1 -

<img width="2246" height="1375" alt="Image" src="https://github.com/user-attachments/assets/c1d1aabd-aaf2-469e-a9c4-de2fadeee8cf" />

Accessibility Issue 2 -

<img width="2027" height="1292" alt="Image" src="https://github.com/user-attachments/assets/328c74c4-922f-453b-a775-f4a103b61831" />

**Additional context**
